### ISSUE TITLE
Android Support

### ISSUE THREAD

**Issue by ronjunevaldoz** [fabmax/kool#22]
Hi @fabmax any updates on android support? looking forward to test this library for my MMO project

**Comment by fabmax** [comment]
Not really. It should still be relatively easy to get that going but unfortunately I currently don't have the time to do that. You are very welcome to contribute in case you want to try 😄 
It should be possible to recycle most of the OpenGL code of the JVM platform and swap LWJGL with Androids [OpenGL bindings](https://developer.android.com/reference/android/opengl/GLES32). The annoying part probably is to write the Android specific stuff (lifecycle management, etc.)
